packages/storybook/stories/MessageThread/MessageThread.stories.tsx

```diff
@@ -298,7 +299,10 @@ const MessageThreadStory = (args): JSX.Element => {
 
   const onLoadPreviousMessages = async (): Promise<boolean> => {
```

Comment:
Can this be called multiple times before we have all the messages retrieved?
I thought I recall that the messages can come in pages and that the promise should return a falsey value to decide if more are needed?

Reply:
Ah yes, this is the storybook. Nm!